packages/framework/fluid-framework/src/index.ts

```diff
@@ -56,6 +56,8 @@ export type {
 	FluidObjectProviderKeys, // Used by FluidObject
 } from "@fluidframework/core-interfaces";
 
+export type { isFluidHandle } from "@fluidframework/runtime-utils";
```

Comment:
This is a new export source for fluid-framework. @Josmithr do we need to make other config changes to ensure the docs are built correctly?

Reply:
Nope :) API-Extractor in this package is configured to bundle all `@fluidframework` packages automatically (assuming they are correctly declared as a direct dependency, which this now is).

Reply:
In this case you can tell because the actual declaration ends up in the API reports in this package, rather than appearing as a re-export from an external package.